querybook/server/logic/metastore.py

```diff
@@ -1,26 +1,25 @@
 import datetime
-from sqlalchemy import func, and_
-from sqlalchemy.orm import aliased
 
 from app.db import with_session
 from const.elasticsearch import ElasticsearchItem
 from lib.sqlalchemy import update_model_fields
-from lib.query_analysis.lineage import process_query
 from models.metastore import (
     DataSchema,
     DataTable,
     DataTableInformation,
     DataTableColumn,
     DataTableOwnership,
-    DataTableQueryExecution,
     DataJobMetadata,
-    TableLineage,
     DataTableStatistics,
     DataTableColumnStatistics,
 )
-from models.query_execution import QueryExecution
 from tasks.sync_elasticsearch import sync_elasticsearch
 
+from env import QuerybookSettings
+
+from lib.utils.plugin import import_plugin
+
+lineage = import_plugin(QuerybookSettings.DATA_LINEAGE_BACKEND)
```

Comment:
can you move the lineage plugin import to another folder? (preferrablely under lib/). So if we want to use lineage we don't have to import logic.metastore

Reply:
Moved the import to a file under lib.